### PR TITLE
Feat/new version

### PR DESCRIPTION
**Pull request by thanh-nguyen-dang**
Jira Ticket: [COV-832](https://occ-data.atlassian.net/browse/COV-832)
- [ ] Remove this line if you've changed the title to (PXP-xxxx): <title>

### CODE REVIEW

File: cli/develop.js [hunk 1]
```diff
@@ -92,16 +95,16 @@ const run = (args) => {
     utils.log("---------------------------------------------------\n\n");
   }).on('error', (error) => {
     if (error.code === 'EADDRINUSE') {
-      utils.error(esapi.encoder.encodeForJavaScript(`Port ${app.get('port')} is currently in use by another program.
-      You must either close that program or specify a different port by setting the shell variable "$PORT". Note that on MacOS / Linux ${chalk.yellow(`lsof -n -i :${app.get('port')} | grep LISTEN`)} should identify the process currently using the port.`));
+      utils.error(`Port ${app.get('port')} is currently in use by another program.
```
Comment (paulineribeyre) [inline cli/develop.js:98]: all the `esapi.encoder.encodeForJavaScript` statements were added by us to fix a security scan warning. It looks like you undid our security changes

File: src/util/globals.js [hunk 2]
```diff
@@ -218,7 +218,7 @@ export const getMapTilesSettings = () => {
     return getExtension("mapTiles");
   }
   /* defaults */
-  const api = 'https://api.mapbox.com/styles/v1/trvrb/ciu03v244002o2in5hlm3q6w2/tiles/256/{z}/{x}/{y}?access_token=pk.eyJ1IjoidHJ2cmIiLCJhIjoiY2tqcnM5bXIxMWV1eTJzazN2YXVrODVnaiJ9.7iPttR9a_W7zuYlUCfrz6A';
+  const api = 'https://api.mapbox.com/styles/v1/trvrb/ciu03v244002o2in5hlm3q6w2/tiles/256/{z}/{x}/{y}?access_token=pk.eyJ1IjoidHJ2cmIiLCJhIjoiY2l1MDRoMzg5MDEwbjJvcXBpNnUxMXdwbCJ9.PMqX7vgORuXLXxtI3wISjw';
```
Comment (paulineribeyre) [inline src/util/globals.js:221]: what is this key? is it the same key that we're using for other maps? if so, let's set it up as a secret/env var like for the other maps?

File: cli/develop.js (hunk 1 above)
Comment (thanh-nguyen-dang) [reply cli/develop.js:98]: @atharvar28 removed this in different other places.. I just applied the same thing

File: src/util/globals.js (hunk 2 above)
Comment (thanh-nguyen-dang) [reply src/util/globals.js:221]: this is the key from nextstraint

Comment (paulineribeyre) [reply src/util/globals.js:221]: we shouldn't be using their public key, we should use ours

Comment (paulineribeyre) [reply src/util/globals.js:221]: see how it's set up for portal: https://github.com/uc-cdis/cloud-automation/blob/1f9cbb52cecf449a5ad76f984fe0dda449d924d3/kube/services/portal/portal-deploy.yaml#L162-L168

**Review by atharvar28 — APPROVED** [review]

**Pull request merged**